app/models/plan.rb

```diff
@@ -184,7 +184,7 @@ def reorder!(account, ids)
   end
 
   def reset_contracts_counter
-    self.class.reset_counters(id, :contracts)
+    Contract.counter_culture_fix_counts(where: { id: id })
```

Comment:
Shouldn't it be 
```suggestion
    Contract.counter_culture_fix_counts(where: { id: id }, only: :contracts)
```

Reply:
there's only one counter cache specified, but true, it won't hurt.

Reply:
So leave it :)